### ISSUE TITLE
Fast access to fields

### ISSUE THREAD

**Issue by jceb** [ivan-tkatchev/tab#2]
Usually, i have to access a certain field in an input stream in order to compute something.  In `tab` I found this to be a bit difficult as I have to split the current line first into separate fields using `cut` and only then I can continue processing it.  Is there a short-cut available like `@` that would allow me to access a field, for example `@[0-9]*`?  The separator could be supplied on the command line like it is done with `awk -F,`.

**Comment by ivan-tkatchev** [comment]
Not yet, but I'm definitely adding something like this in the next release.

`@[0]` looks like a neat way to do this, I like it.

Specifying the delimiter as a command-line switch seems inelegant but it's a working solution.

Another possibility is a version of `cut` that works on sequences of strings, so instead of 

```
    [cut(@,"\t",0), cut(@,"\t",1)]
```

you could do

```
    [ @~0, @~1 : cut(@,"\t") ]
```

**Comment by ivan-tkatchev** [comment]
In the end I decided to go with the second option. Fixed in master and 5.0.
